packages/clients/meldemichel/src/plugins/Gfi/Feature.vue

```diff
@@ -0,0 +1,168 @@
+                {{ $t(`meldemichel.gfi.${item}`) }}
+              </td>
+              <!-- eslint-disable-next-line vue/no-v-html -->
+              <td
+                v-html="$t(formatProperty(item, currentProperties[item]))"
+              ></td>
+            </tr>
+          </tbody>
+        </template>
+      </v-simple-table>
+    </v-card-text>
+    <v-card-title>{{ $t(`meldemichel.gfi.beschr`) }}</v-card-title>
+    <v-card-text>{{ currentProperties['beschr'] }}</v-card-text>
+    <v-card-title>{{ $t(`meldemichel.gfi.rueck`) }}</v-card-title>
+    <v-card-text>{{ currentProperties['rueck'] }}</v-card-text>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue, { PropType } from 'vue'
+import { GeoJsonProperties } from 'geojson'
+import { mapActions, mapMutations, mapGetters } from 'vuex'
+
+type GfiIndexStep = -1 | 1
+
+export default Vue.extend({
+  name: 'MeldemichelGfiFeature',
+  props: {
+    currentProperties: {
+      type: Object as PropType<GeoJsonProperties>,
+      required: true,
+    },
+    clientWidth: {
+      type: Number,
+      required: true,
+    },
+    exportProperty: {
+      type: String,
+      default: '',
+    },
+    showSwitchButtons: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  data: () => ({
+    infoFields: ['skat', 'start', 'statu'],
+  }),
+  computed: {
+    ...mapGetters(['hasSmallWidth', 'hasWindowSize']),
+    ...mapGetters('plugin/gfi', [
+      'windowFeatures',
+      'visibleWindowFeatureIndex',
+    ]),
+    displayImage(): boolean {
+      return this.currentProperties.pic
+    },
+  },
+  methods: {
+    ...mapMutations('plugin/gfi', ['setVisibleWindowFeatureIndex']),
+    ...mapActions('plugin/gfi', ['close']),
+    formatProperty(type: string, value: string): string {
+      if (!value) {
+        return ''
+      }
+      if (type === 'skat') {
+        return `meldemichel.skat.${value}`
+      }
+      if (type === 'start') {
+        return `${value.substring(6, 8)}.${value.substring(
+          4,
+          6
+        )}.${value.substring(0, 4)}`
+      }
+      return value
+    },
+    resize(): void {
+      window.dispatchEvent(new Event('resize'))
+    },
+    /** switch to next or previous feature */
+    switchFeature(by: GfiIndexStep): void {
+      const {
+        visibleWindowFeatureIndex,
+        windowFeatures,
+        setVisibleWindowFeatureIndex,
+      } = this
+      const maxIndex = windowFeatures.length - 1
+      const nextIndex = visibleWindowFeatureIndex + by
+      if (nextIndex < 0) {
+        setVisibleWindowFeatureIndex(windowFeatures.length - 1)
+      } else if (nextIndex > maxIndex) {
+        setVisibleWindowFeatureIndex(0)
+      } else {
+        setVisibleWindowFeatureIndex(nextIndex)
+      }
+    },
+  },
+})
+</script>
+
+<style lang="scss" scoped>
+* {
+  color: #003064 !important;
+}
+
+.meldemichel-gfi-grip-icon {
+  left: 50%;
+  transition: translateX(-50%);
+}
+
+.meldemichel-gfi-card {
+  min-width: 20vw;
+}
```

Comment:
apply suggestion "remove min width"

Reply:
It makes GFI windows wider, should they be narrower. Pretty much exactly what you probably assumed it would do. I removed it nonetheless, we'll see if we need something like this later on productive data.